manifests/params.pp

```diff
@@ -176,13 +188,19 @@
 
   # Passenger config
   $server_app_root = "${dir}/rack"
-  $server_ssl_dir  = "${vardir}/ssl"
+  $server_ssl_dir  = versioncmp($::puppetversion, '4.0') ? {
```

Comment:
What is it for FreeBSD? In the definition we use above it's the same as Linux so maybe that should be fixed instead.